### ISSUE TITLE
[FEATURE] Source from https://preview.tabler.io/ in version 1.0.0-alpha.7

### ISSUE THREAD

**Issue by MickaelTH** [tabler/tabler#727]
**Is your feature request related to a problem? Please describe.**
Since the version **1.0.0-alpha.14** and the switch from **tabler/tabler** to **@tabler/core**, I have a lot of breaking changes that need time to get resolved (JS conflict with other third party script such as _fancytree_ for example), so it will take a while until I manage to fully move to v1.0.0.

One thing that was extremely useful, and saved me a lot of time, was the ability to check the [https://preview.tabler.io/](https://preview.tabler.io/) website to grab some HTML directly.

It was on **1.0.0-alpha.7** for few months, but now it has been updated to **1.0.0-alpha.14** so I have lost all HTML references and I cannot update to 1.0.0-alpha.14 (yet) because of conflicts.

**Describe the solution you'd like**
Kindly provide the HTML sources files of the https://preview.tabler.io/ website with each version.

**Describe alternatives you've considered**
Only provide the sources from https://preview.tabler.io/ for version 1.0.0-alpha.7 as i suspect a lot of users are still under this version and will until the version 1.0.0. 

Really appreciate if this could be done, thanks.

**Comment by colorninja** [comment]
You can always download tag 1.0.0-alpha.7 and compile them yourself.
```
git clone https://github.com/tabler/tabler.git
cd tabler
git checkout 1.0.0-alpha.7
```

Then you can follow the instructions to setup environment and build locally.

Let me know if that works.

**Comment by MickaelTH** [comment]
Hi @colorninja 

I understand and I tried that already but I am facing a lot of errors with the build that are above my skills to resolve such as:

`Gem::Ext::BuildError: ERROR: Failed to build gem native extension.`

`An error occurred while installing eventmachine (1.2.7), and Bundler cannot continue.`

`ERROR:  Error installing eventmachine:
        ERROR: Failed to build gem native extension.`

I spent hours trying to figure this out but I just stopped because I don't have the skills for that. Thus my feature request to avoid all this headache trying to figure out how to build it.

**Comment by FreexD** [comment]
@MickaelTH already compiled HTMLs are located in `/demo` folder so you should be able to see them in the browser. Tabler dist files (like CSS, JS, SCSS sources) are in `/dist` folder in case you need them as well. 

However the point is made and tabler could use either a migration guide or a way of viewing old versions.

**Comment by MickaelTH** [comment]
Hi @FreexD 

Thanks for your reply. I must have been blind because the files are in the /demo folder indeed:

![image](https://user-images.githubusercontent.com/26375917/100345266-ab848500-3014-11eb-815d-446c98fbef82.png)

Thank you for this. Also, it would be nice to keep those online in **https://preview.tabler.io/{version}** for reference (unless it is too much work to keep history).

I understand this is still alpha and a lot will changes in the next weeks/months - but it could help some users to have this to keep slowly keep the pace.

Thanks again!

**Comment by DivyanshuShukla001** [comment]
Anybody can tell how to use github Iam new

**Comment by FreexD** [comment]
@DivyanshuShukla001 I'm not sure if you're in the right place ;) Can you explain how does that relate to tabler?

**Comment by Oanko** [comment]
@FreexD ,would you help me?
i just downloaded 1.0.0-alpha.15 as zip as usual from github, then after extracting them, js (i think !!) aren't working,
haven't done anything,just uploaded that downloaded zip to server and extracted,but on there on pages, aren't expanding the nav,or opening modal but all css ,js are linked perfectly ,as i did nothing..and via source code it's all good ...
i tested locally as well but it's same...
mirror : http://abulia.design/tabler-dev/demo/index.html

**Comment by FreexD** [comment]
Hey @Oanko, I've released a new version `1.0.0-alpha.16`. Bootstrap has changed their data attributes to be prefixed by `data-bs-*` as of the newest version and that broke the JS components on tabler. Everything should work fine with the new version ;)

**Comment by Oanko** [comment]
@FreexD ,thanks for quick response...
for now it's working as a charm,if something gets wrong ,you should get another error report to delt with.... ;->
